pom.xml

```diff
@@ -12,12 +12,13 @@
     <artifactId>remoting-kafka-plugin</artifactId>
     <name>Remoting Kafka Plugin</name>
     <description>Allows users to start agent using Kafka</description>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0.0-alpha-1-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <properties>
         <remoting.version>3.22</remoting.version>
         <jenkins.version>2.129</jenkins.version>
+        <kafka-client-lib.version>1.0.0-alpha-1-SNAPSHOT</kafka-client-lib.version>
```

Comment:
i doubt that Maven Release plugin will be able to handle it correctly. I would recommend to avoid such property, the versions will be resolved automatically during the multi-module release if you just use dependencies

Reply:
@oleg-nenashev my IDE cannot resolve this and I also cannot `mvn clean install` the project when I delete the property from the parent POM.

Reply:
Or if we want to use the same version for all the POMs, we can set a global version properties in the parent POM and reuse it, WDYT?